validator/sawtooth_validator/networking/dispatch.py

```diff
@@ -127,16 +137,21 @@ def remove_send_last_message(self, connection):
                          connection)
 
     def dispatch(self, connection, message, connection_id):
-        if message.message_type in self._msg_type_handlers:
-            priority = self._priority.get(message.message_type, Priority.LOW)
+        message_type = message.message_type
+        if message_type in self._msg_type_handlers:
+            priority = self._priority.get(message_type, Priority.LOW)
```

Comment:
This change is non-functional and not related to the commit message. Should be in a separate commit or dropped.